playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Ensure kubectl is properly configured
+  include_role:
+    name: common-deployer
+    tasks_from: kubectl-setup
+
+- name: Add soc extra repositories
```

Comment:
nit: should that be `Add SES extra repositories` ?

Reply:
:+1:

Reply:
Still not fixed but can be done later...